package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "node.js-mentoring-program",
+  "version": "1.0.0",
+  "description": "<img src=\"https://jsremotely.com/static/icons/nodejs.png\" width=\"100px\"/>",
+  "main": "index.js",
+  "scripts": {
+    "task1": "nodemon --exec babel-node ./hw-1/task1.js -I",
+    "task2": "nodemon --exec babel-node ./hw-1/task2.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
There should be a 3rd task with babel

Reply:
I was doing it task by task. I thought that after step 3 both programs have to use babel. As they do finally. Here's the list of commits: https://github.com/interist32/node.js-mentoring-program/pull/1/commits

Reply:
ok, got it. can be accepted, but it was expected to run 2nd module with babel in seperate run script